components/gcp/bigquery/query/README.md

```diff
@@ -110,3 +154,10 @@ experiment = client.create_experiment(EXPERIMENT_NAME)
 run_name = pipeline_func.__name__ + ' run'
 run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename, arguments)
 ```
+
+#### Inspect the output
+
+
+```python
```

Comment:
this is not python. So maybe use bash?

Reply:
This is automatically added by Jupyter notebook which is a python notebook. The command here is a magic command which internally calls python subprocess API. So basically, it's still a python code.

Reply:
Got it.